ex03.ex

```diff
@@ -153,8 +183,18 @@ defmodule Ex03 do
   Think a little about a nice way to lay this code out.
   """
 
-  def won . . . "your code"
+  def won( { a, a, a,  _, _, _,  _, _, _ } ), do: a
```

Comment:
# -1

I was hoping for a 2d layout, where each function head looked like an actual board.